tests/bears/BearTest.py

```diff
@@ -105,7 +116,7 @@ def tearDown(self):
     def test_simple_api(self):
         self.assertRaises(TypeError, TestBear, self.settings, 2)
         self.assertRaises(TypeError, TestBear, None, self.queue)
-        self.assertRaises(NotImplementedError, self.uut.kind)
+        self.assertEqual(BEAR_KIND.GLOBAL, self.uut.kind())
```

Comment:
please create a new bear for it without an implementation for `kind`, this one explicitly tests the default behaviour^^